source/globalCommands.py

```diff
@@ -188,7 +188,7 @@ def script_cycleAudioDuckingMode(self, gesture):
 	def script_toggleInputHelp(self, gesture):
 		inputCore.manager.isInputHelpActive = not inputCore.manager.isInputHelpActive
 		# Translators: This will be presented when the input help is toggled.
-		stateOn = _("input help on")
+		stateOn = _("input help on. Press this command again to turn it off.")
```

Comment:
I think this should announce the command itself
```suggestion
		stateOn = _("input help on. Press NVDA+1 again to turn it off.")
```

Reply:
What to do if the user changes the shortcut keys

Reply:
This should work, but further testing is needed.

```suggestion
		stateOn = _("input help on. Press {gestureKeys} again to turn it off.").format(gestureKeys=gesture.displayName)
```